task-3_mongoDB-p1/README.md

```diff
@@ -0,0 +1,92 @@
+```
+mongoimport --db frontcamp --collection restaurants --file C:/data/db/restaurants.json
+```
+
+# task 3:
+
+1) RESULT: 728
+---------------
+
+```
+{ borough: "Queens", cuisine: "Chinese"}
+```
+
+2) RESULT: 5dc468ae4c745d010cdeb463 "restaurant_id": "40372466"
+---------------
+
+```
+$project {
+  _id: "$_id",
+  maxScore: {$max: "$grades.score"}
+} + $sort {
+  maxScore: -1
+}
+```
+
+3) Done
+---------------
+```
+{ borough: "Manhattan"}, { $set: { "grades": { grade : "A", score: 7, date: ISODate() }}}
+```
+
+4) RESULT: name:"Silver Krust West Indian Restaurant"
+---------------
```

Comment:
I guess these are not all names

Reply:
Collection includes only 3 items with index 8 ({"grades.8": {$exists: true}}) but only one has grade less that 7. Or maybe I something don't understand?

Reply:
I recommend redo the query, the answer isn't entire

Reply:
I used query such as { cuisine: "Seafood", "grades.grade": "B", "grades.date": {$gte: ISODate("2014-02-01T00:00:00.000Z"), $lt: ISODate("2014-03-01T00:00:00.000Z")}}

Reply:
I used query such as { "grades.8.score": {$lt: 7}}

Reply:
Use something like as `>db.restaurants.distinct("name", query)`

Reply:
![image](https://user-images.githubusercontent.com/45118964/68549001-650db980-0404-11ea-9215-3c6f0624efdd.png)